.travis.yml

```diff
@@ -11,20 +11,20 @@ branches:
 language: generic
 
 env:
-  - NPM_TAG=latest IMAGE_NAME=theia-full
-  - NPM_TAG=next IMAGE_NAME=theia-full
-  - NPM_TAG=latest IMAGE_NAME=theia
-  - NPM_TAG=next IMAGE_NAME=theia
-  - NPM_TAG=latest IMAGE_NAME=theia-java
-  - NPM_TAG=next IMAGE_NAME=theia-java
-  - NPM_TAG=latest IMAGE_NAME=theia-go
-  - NPM_TAG=next IMAGE_NAME=theia-go
-  - NPM_TAG=latest IMAGE_NAME=yangster
-  - NPM_TAG=next IMAGE_NAME=yangster
-  - NPM_TAG=latest IMAGE_NAME=theia-python
-  - NPM_TAG=next IMAGE_NAME=theia-python
-  - NPM_TAG=latest IMAGE_NAME=theia-swift
-  - NPM_TAG=next IMAGE_NAME=theia-swift
+  - NPM_TAG=latest IMAGE_NAME=theia-full NODE_VERSION=8.14.1
```

Comment:
Why do you pin only theia-full to a specific micro version and not the others?

Reply:
i've just mimicked what was in images before. Should i use latest node for full as well? Not sure what was the reasoning behind.

Reply:
I do not think there is a need for that specific version to be pinned - the various images tend to be updated individually, since some already have Node.js, but others not (i.e. it's hard to be uniform).

Reply:
I've removed pinning

Reply:
Sorry, I was wrong: pinning was correct, since we use that version to fetch a specific tarball from `nodeja.org`, which fails when we only request version `8`. (seems to work for version `10` however)

Reply:
We could however pin to a different version, like the more recent `8.15.1`